_build_dependencies.sh

```diff
@@ -8,4 +8,4 @@ export PHP_VERSION=5.6
 # Byond Major
 export BYOND_MAJOR=513
 # Byond Minor
-export BYOND_MINOR=1514
+export BYOND_MINOR=1526
```

Comment:
This was changed to `1526` from `1514` because landmarks will not initialize on that byond version. Dont ask.